javers-core/src/test/groovy/org/javers/core/metamodel/type/TypeMapperTest.groovy

```diff
@@ -279,6 +279,7 @@ class TypeMapperTest extends Specification {
     @Unroll
     def "should ignore all props of subclass when superclass #classType has Included properties definition"(){
         given:
+        def mapper = JaversTestBuilder.javersTestAssembly().typeMapper
```

Comment:
The new instance for each internal test case was initially necessary because I temporarily made TypeMapperEngine throw on second registration attempt.
The TypeMapperEngine in this branch will ignore the second attempt of registering a definition for PropsClass and the test would pass, but more or less by accident, so I thought I leave the local instance here to make it clear, that the internal test cases expect fresh instances.

Just let me know if I should restore this test to its original state.